recipes/tinyarray/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "tinyarray" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://gitlab.kwant-project.org/basnijholt/tinyarray/repository/archive.tar.gz?ref=v{{ version }}
+  sha256: 83ae0dad1d352128083db1e89242197673b0578b5b827b882fbd23094e254fbb
+  
+build:
+  # skip: true  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+  run:
+    - python
+
+test:
+  requires:
+    - nose
+  source_files:
+    - test_tinyarray.py
+  commands:
+    - $PYTHON test_tinyarray.py   # [unix]
```

Comment:
OK apparently Jinja doesn't like that, so just merge those two lines into a single one `python test_tinyarray.py`  - sorry!